openapi.yaml

```diff
@@ -4161,14 +4164,25 @@ components:
           items:
             $ref: '#/components/schemas/Organization'
           type: array
+        datasources:
+          description: The list of datasources configured for the account.
+          items:
+            $ref: '#/components/schemas/Datasource'
+          type: array
         jira:
           $ref: '#/components/schemas/JIRAInstallation'
       required:
       - admins
       - organizations
       - regulars
+      - datasources
       type: object
     # yamllint disable-line rule:key-ordering
+    Datasource:
+      description: A datasource configured for the account.
+      enum:
+      - github
+      - jira
```

Comment:
what about CI, Gaetano? Isn't it also one of the data sources for us? Afaik it doesn't have to be always GHA, I think we also support CircleCI and Jenkins. But I might be wrong here, just double-checking.

Reply:
If I'm not wrong we support the checks tracked by GitHub, so yes we support also external CI configured in GitHub by the repo owner.
My proposal anyway is to add `deployments` and `code-check` to this enum when we'll need and implement them.